doc/src/conf.py

```diff
@@ -220,6 +308,7 @@
 # SymPy logo on title page
 html_logo = '_static/sympylogo.png'
 latex_logo = '_static/sympylogo_big.png'
+html_favicon = '../_build/logo/sympy-notailtext-favicon.ico'
```

Comment:
This should likely be logo/... instead? Otherwise, you won't get the logo on the first build (unless I'm misunderstanding your build process).

Reply:
Yeah, we compile the logo from svg to png and ico as part of the build process (the Makefile dependencies ensure it's compiled first).